docs/source/asr/speaker_diarization/datasets.rst

```diff
@@ -213,52 +210,3 @@ The following are descriptions about each field in an input manifest JSON file.
   
    TS3012d.Mix-Headset 1 12.879 0.32 okay NA lex MTD046ID
    TS3012d.Mix-Headset 1 13.203 0.24 yeah NA lex MTD046ID
-
-
-Evaluation on Benchmark Datasets
```

Comment:
Should we keep a list of benchmark datasets here and maybe a table showcasing the performance of our models on these sets compared to current SOTA?